changes/5716.added

```diff
@@ -1,2 +1,4 @@
 Added CloudNetwork model and REST API.
-Added CloudNetwork to CircuitTermination model and REST API.
\ No newline at end of file
+Added CloudNetwork to CircuitTermination model and REST API.
+Add CloudNetwork UI.
```

Comment:
Should probably combine this with the first line.
Something like:
Added CloudNetwork Model, UI, GraphQL and REST API.